include/Metabox/Posts/Tagtocategory.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Bulk Move Posts Tag To Category Metabox.
+ *
+ * @since 2.0.0
+ */
+class BM_Metabox_Posts_Tagtocategory extends BM_Metabox_Base {
+
+    protected function initialize() {
+        $this->meta_box_slug         = 'bm-posts-tag-to-category';
+        $this->messages['box_label'] = __( 'Move Posts Tag To Category', 'bulk-move' );
```

Comment:
Change label to `Move Posts from Tag to Category`

Reply:
Fixed